Mathlib/Data/Finset/Basic.lean

```diff
@@ -2665,6 +2665,7 @@ theorem filter_congr {s : Finset α} (H : ∀ x ∈ s, p x ↔ q x) : filter p s
 
 variable (p q)
 
+@[simp]
```

Comment:
```suggestion
```
It looks like this is solved `by simp`. It would nice to be able to trust the linter.